tests/endpoints/echo/client/client.go

```diff
@@ -197,3 +205,44 @@ func DoCorsPreflightRequest(url, origin, requestMethod, requestHeader, referer s
 	defer resp.Body.Close()
 	return resp.Header, nil
 }
+
+// DoHttpsGet performs a HTTPS Get request to a specified url
+func DoHttpsGet(url string, httpVersion int, certPath, keyPath string) ([]byte, error) {
```

Comment:
Maybe set enum for httpVersion

Reply:
I think it is not necessary, enum doesn't give any benefit.